### PR TITLE
Explain that `/job-sets` requires a queue name

### PR DESCRIPTION
**Pull request by zuqq**
Consider this programmer art; it's just a sentence in the middle of the screen:

image

### CODE REVIEW

File: internal/lookout/ui/src/containers/JobSetsContainer.tsx [hunk 1]
```diff
@@ -229,6 +229,9 @@ class JobSetsContainer extends React.Component<JobSetsContainerProps, JobSetsCon
   }
 
   private async loadJobSets() {
+    if (this.state.queue === "") {
+      return
+    }
```
Comment (zuqq) [inline internal/lookout/ui/src/containers/JobSetsContainer.tsx:234]: This is a drive-by change that prevents us from making requests if no queue name is given.

**Review by severinson — APPROVED** [review]
